tests/minus.rs

```diff
@@ -0,0 +1,18 @@
+#[cfg(test)]
+mod tests {
+    use lrlex::lrlex_mod;
+    use lrpar::lrpar_mod;
+    use yaiwr::eval;
+    lrlex_mod!("calc.l");
+    lrpar_mod!("calc.y");
+    #[test]
+    fn eval_minius_exp() {
+        let lexerdef = calc_l::lexerdef();
+        let input = "1-1".to_string();
```

Comment:
You may be able to remove the `.to_string`, as `eval` want to accept a referenced string and string literals are of type `&'static str`.

Reply:
done